### DISCUSSION TITLE
cannot select teams with "Allow specified actors to bypass required pull requests" option

### DISCUSSION THREAD

**Discussion by Macmee** [community/community#49027] [Pull Requests]
### Select Topic Area

Bug

### Body

We are trying to use "Allow specified actors to bypass required pull requests" to allow our iOS-team to bypass a merge rule on a protected branch, however you cannot select teams using the input, you can only select users despite it saying you can select teams here in the subheading "Specify people, teams, or apps who are allowed to bypass required pull requests."

**Comment by Anmol-Baranwal** [comment]
You can read the docs [here](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule)

I believe that it needs to be a organization on GitHub, so its members are allowed to bypass. I hope your doubts are resolved.
It is definitely not a bug, since it is used by several thousand members.

**Reply by Macmee** [reply]
Hi friend, we are an enterprise organization  on GitHub and have the team setup, but still it won't let us search for anything besides individual members

**Reply by peresypkinamarina** [reply]
Experienced the same issue. Fixed by provide team Write access to repo

**Reply by pcolby** [reply]
> Experienced the same issue. Fixed by provide team Write access to repo

Thanks @peresypkinamarina, I have no idea how long it would have taken me to figure that out (stupid lack of feedback from GitHub's UI 🙄)
